test/unit_overapproximate.jl

```diff
@@ -194,4 +216,15 @@ for N in [Float64] # due to sparse vectors: a = sparse(Float32[1 -1; 1 1];); a \
     Z1 = overapproximate(vTM, Zonotope)
     @test center(Z1) == N[3, -2.5]
     @test Matrix(generators(Z1)) == N[1 -1 0; -1 0 0.5]
+
+    # decomposed intersection between cartesian product array and abstract polyhedron
+    i1 = Interval(N[0, 1])
+    i2 = Interval(N[2, 3])
+    h1 = Hyperrectangle(low=N[3, 4], high=N[5, 7])
+    cpa = CartesianProductArray([i1, i2, h1])
+    G_comb = HPolyhedron([HalfSpace(N[1, 1, 0, 0], N(2.5))])
+    int_g_comb = Intersection(cpa1, G_comb)
+    o_d_int_g_comb = overapproximate(int_g_comb, CartesianProductArray, Hyperrectangle)
+    o_bd_int_g_comb = overapproximate(int_g_comb, BoxDirections)
+    @test overapproximate(o_d_int_g_comb) ≈ overapproximate(o_bd_int_g_comb)
```

Comment:
```suggestion
    @test overapproximate(o_d_int_g_comb) ≈ overapproximate(o_bd_int_g_comb)
    projection = project(o_bd_int_g_comb, [1, 2], OctDirections)
    @test vertices_list(projection) ≈ [[0., 2.5], [0., 2.], [0.5, 2.]]
```

Reply:
Projection of `x+y<=n ∩ Hyperrectangle` onto *2 blocks* x and y won't be a triangel, it will be two intervals, so the result will be an interval hull or, in this case, `Hyperrectangle`

Reply:
Indeed, I had locally reassigned the variable. I meant this:
```julia
projection = project(overapproximate(int_g_comb, OctDirections), [1, 2], OctDirections)
```